docs/site/tutorials/protocol_oriented_generics.ipynb

```diff
@@ -0,0 +1,1141 @@
+{
```

Comment:
<p>1) What does "very expressive" mean? How does one quantify expressiveness?</p><p><br></p><p>2) Would "the combined power of protocol oriented programming and generics" change meaning if it said "the combined power of protocols and generics"? If no, what is the difference between protocol-oriented programming and simply using protocols?</p><p><br></p><p>3) Which of these options at our disposal is/are (the most) idiomatic? It would be good to provide some guidance to promote consistency in future S4TF codebases.</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/swift/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>